### PR TITLE
Add LangChain RAG notebook

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
Markdown:

MicrosoftTeams-image

CSV:

image

### CODE REVIEW

File: notebooks/272-langchain-rag/272-langchain-rag.ipynb [hunk 1]
```diff
@@ -0,0 +1,869 @@
+{
```
Comment (eaidova) [inline notebooks/272-langchain-rag/272-langchain-rag.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    input_info = [("input_ids", input_shape, np.int64),
</pre><p>input info is not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by OpenVINO-dev-contest** [comment]
Not support for Chinese LLM yet

File: notebooks/254-llm-chatbot/254-rag-chatbot.ipynb [hunk 2]
```diff
@@ -0,0 +1,1286 @@
+{
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/254-rag-chatbot.ipynb:1]: <p>if you rebase notebooks, probably you can see also tiny-llama-1b model. I think it is good to include it too, as it is small and fast and does not required a lot of RAM, I see you have it in widget (it is stored in your cell output), but have not in description</p><p> </p><p>For embeddings models, please provide links on some sources (sentence-transofmers, CoSENT and model cards)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (eaidova) [inline notebooks/254-llm-chatbot/254-rag-chatbot.ipynb:1]: <p>I think in future, we can have more embeddings models, it maybe not good choice to differ them only by language (e.g. having names). I think it is better to convert LLM, then embeddings model and add choice of Chinese if model support Chinese?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Comment by eaidova** [comment]
@OpenVINO-dev-contest, I stayed some small comments. Could you please also include notebook description in readme and also possibly in main readme in tranding notebooks section

**Comment by OpenVINO-dev-contest** [comment]
> @OpenVINO-dev-contest, I stayed some small comments. Could you please also include notebook description in readme and also possibly in main readme in tranding notebooks section

done

File: notebooks/254-llm-chatbot/254-rag-chatbot.ipynb (hunk 2 above)
Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-rag-chatbot.ipynb:1]: done

**Review by eaidova — APPROVED** [review]

**Pull request merged**